registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties

```diff
@@ -1,4 +1,4 @@
-spring.cloud.config.uri=http://localhost:51000/config
+spring.cloud.config.uri=localhost
```

Comment:
Is this change necessary ?

Reply:
Yes. these are Default properties. we change it http://localhost:51000/config when we setup in our local system.